### DISCUSSION TITLE
Is the support for Vercel middleware + Sveltekit?

### DISCUSSION THREAD

**Discussion by alexbjorlig** [sveltejs/kit#10130] [Q&A] [answered]
Some questions, that I could not find an answer for:

1. Does the Sveltekit Vercel adapter support middleware?
2. Does it work with `--prebuilt` deployments from the Vercel CLI?
3. Are there any plans to support features like this when running `dev`/`preview`?

Thanks in advance 🤗

**Comment by manlito** [comment, marked as answer]
From Vercel
> We recommend [using SvelteKit's server hooks](https://kit.svelte.dev/docs/hooks) to modify responses. Due to SvelteKit's client-side rendering, you cannot use Vercel's Edge Middleware with SvelteKit

Source: https://vercel.com/docs/frameworks/sveltekit#edge-middleware
